frontend/src/app/assess/[projectId]/premium-features/components/FeatureCard.tsx

```
@@ -0,0 +1,57 @@
+    const handleClick = () => router.push(href);
+
+    const handleKeyDown = (e: KeyboardEvent<HTMLDivElement>) => {
+        if (e.key === "Enter" || e.key === " ") {
+            e.preventDefault();
+            handleClick();
+        }
+    };
+
+    return (
+        <motion.div
+            role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)